examples/behaviors/basic/actions/replace-inner/nested/_replace_fragment_inner2.xml.njk

```diff
@@ -0,0 +1,3 @@
+{% from 'macros/button/index.xml.njk' import button %}
+
+{{ button('Press me again...', {"action": "replace-inner", "href":"/behaviors/basic/actions/replace-inner/nested/_replace_fragment_inner2.xml", "xmlns":"https://hyperview.org/hyperview"}) }}
```

Comment:
I think we should make the behavior an explicit XML element here - separating the UI from the "content"

Reply:
I added this to demonstrate the idea that button can have the behavior attributes as opposed to passing a <behavior/> explicitly. I will revert this if seems unecessary to the user

Reply:
passing <behavior/> as per our discussion